webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -390,6 +397,27 @@ export class SakaiRubric extends RubricsElement {
       this.cloneRubric(e);
     }
   }
+
+  // SAK-47640 - Get the maximum and minimum possible grade by multiplying the max or min rating by the criterion weight.
+  getMaxPoints(criterions) {
+    return this.getPoints(criterions, Math.max);
+  }
+
+  getMinPoints(criterions) {
+    return this.getPoints(criterions, Math.min);
+  }
+
+  getPoints(criterions, minOrMax) {
+    let totalPoints = 0;
+    criterions
+      .filter( (criterion) => criterion.ratings.length > 0)
+      .forEach( (criterion) => {
+        totalPoints += minOrMax(...criterion.ratings.map( (rating) => {
```

Comment:
You should extract this into a function as you are using it in a few places.

Reply:
Hi, where would you place the new function?

Reply:
I'd put it in sakai-rubrics-utils.js

Reply:
I'm having trouble to successfully call the method once located on `sakai-rubrics-utils.js`.
Is there any special way for this case?

Reply:
You need to import it at the top of sakai-rubric.js, es6 style. Something like:

import { calculatePoints } from "./sakai-rubric-utils.js";

you also need to export it in sakai-rubric-utils.js

Reply:
maybe call the function calculateCriteriaPoints, something like that?